### PR TITLE
yousra nodejs Week2

### PR DESCRIPTION
**Pull request by YousraElmag**

### CODE REVIEW

File: homework/config-files/HackYourTemperature/sources/keys.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ 
+ const API_KEY= '3c478b50b4e7922d2929fcbe0e8f1854'
+export default API_KEY
```
Comment (dardecena) [inline homework/config-files/HackYourTemperature/sources/keys.js:3]: This is not acceptable, as your API KEY is now compromised. Please get new API key credentials, update your keys.js file, and add  the keys.js file to your .gitignore.

File: homework/config-files/HackYourTemperature/app.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    if (response.status === 404) {
+      res.json({ weatherText: "City is not found!" });
+    } else {
+      const temperature = data.main.temp;
+      res.json({ weatherText: `The temperature in ${cityName} is ${temperature}K` });
+    }
+  } catch (error) {
+    res.status(500).json({ error: 'Internal Server Error' });
+  }
+});
```
Comment (dardecena) [inline homework/config-files/HackYourTemperature/app.js:30]: Good use of try/catch, and using the different http status codes, and informative messages.

**Review by dardecena — CHANGES_REQUESTED** [review]
Please read over comments and make the suggested changes. It is good to understand why it's important to keep API keys secret, and how to keep them secret. 

This is a good learning exercise to understand what to do when sensitive information is exposed and how to recover from it.

**Comment by YousraElmag** [comment]
thanks for your reviews i add  keys  to gitignore

File: homework/config-files/HackYourTemperature/.gitignore
Comment (dardecena) [inline homework/config-files/HackYourTemperature/.gitignore:1]: Good job!

**Review by dardecena — APPROVED** [review]
Thanks for making the changes. Looks good!

File: homework/config-files/HackYourTemperature/.gitignore
Comment (YousraElmag) [reply homework/config-files/HackYourTemperature/.gitignore:1]: thanks

**Pull request closed without merging**